build/source/engine/time_utils.f90

```diff
@@ -30,12 +30,12 @@ module time_utils_module
  ! ******************************************************************************************
  ! public subroutine extractTime: extract year/month/day/hour/minute/second from units string
  ! ******************************************************************************************
- subroutine extractTime(refdate,iyyy,im,id,ih,imin,dsec,err,message)
+ subroutine extractTime(refdate,iyyy,im,id,ih,imin,isec,err,message)
```

Comment:
These changes appear unrelated to the addition of the GRU dimension. Can you split them out so that the PR is self-contained?